### PR TITLE
Don't run openpose (mock)

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

**Review by jlunenburg — COMMENTED** [review]
What do we run in simulation?

**Comment by MatthijsBurgh** [comment]
In the past there was a mock, now there is no mock. So the node dies on startup. I don't think we should be running any openpose in simulation. On the robots it is really robot specific, because of the need of gpu.

File: launch/essentials/rgbd_to_ros.launch [hunk 1]
```diff
@@ -39,9 +39,4 @@
 		<!-- <remap from="camera/depth/camera_info" to="top_kinect/depth_registered/sw_registered/camera_info" /> -->
 	</node>
 
-	<!-- Launch a mock of the pose detector in sim -->
-	<group unless="$(optenv ROBOT_REAL false)" ns="pose_detector">
```
Comment (jlunenburg) [inline launch/essentials/rgbd_to_ros.launch:43]: Wat draaien we wel in simulatie?

Comment (MatthijsBurgh) [reply launch/essentials/rgbd_to_ros.launch:43]: Geen openpose

**Review by jlunenburg — APPROVED** [review]

**Pull request merged**